test/libethereum/State.cpp

```diff
@@ -93,6 +93,8 @@ void doStateTests(json_spirit::mValue& _v, bool _fillin)
 }
 } }// Namespace Close
 
+//Disable OLD StateTests. Use GeneralStateTests at StateTests.cpp instead
+/*
```

Comment:
I've disabled it so it wont take execution time on windows build. 
Those tests are now implemented at GeneralStateTests so no need to run it twice.